src/main/scala/is/hail/variant/GenotypeStream.scala

```diff
@@ -116,6 +128,14 @@ object GenotypeStream {
   }
 }
 
+object ToIteratorGT {
+  def apply(it: Iterable[Genotype]): Iterator[Int] =
```

Comment:
It might be clearer to add an implicit from `Iterable[Genotype]` to `RichIterableGenotype` with a `hardcallIterator` member.  Then you can do `gs.hardcallIterator` for `gs: Iterable[Genotype]`.